native-platform/src/test/groovy/net/rubygrapefruit/platform/file/PosixFilesTest.groovy

```diff
@@ -317,6 +317,29 @@ class PosixFilesTest extends FilesTest {
         e.message == "Could not read symlink $symlinkFile: could not readlink (errno 22: Invalid argument)"
     }
 
+    @Unroll
+    def "can get mode for a file behind symbolic link"() {
```

Comment:
:x: We should add a test with a broken symlink, too. We should test both the new method and the old method.

Reply:
I'll add a test for a broken link, ok.

Both implementations are covered by L333-337, which case do you miss

Reply:
They cover it for an non-borken link. We should cover the both old and the new method for a broken link as well.

Reply:
I've added a test.